### PR TITLE
Add a first draft of the Theming doc.

### PR DESCRIPTION
**Pull request by jverkoey**
Pivotal story: https://www.pivotaltracker.com/story/show/156387074

### CODE REVIEW

**Comment by jverkoey** [comment]
Screenshot:

screen shot 2018-04-26 at 10 43 05 pm

File: docs/theming/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+layout: landing
+section: docs
+path: /docs/theming/
+-->
+
+# Theming Material Components
+
+Material Theming is a consistent way to apply a uniform design across your app when using Material Components for iOS.
+
+Material Theming on iOS consists of two primary patterns: schemes and themers.
```
Comment (romoore) [inline docs/theming/README.md]: Nit: Why two separate paragraphs, each a single sentence? Can we combine them?

File: docs/theming/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+**Schemes** represent your design as systemized symbols.
```
Comment (romoore) [inline docs/theming/README.md]: Consider making these a bulleted list.

File: docs/theming/README.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+**Themers** are the glue that apply scheme symbols to components.
+
+For example, there is a scheme for both the Material Design typography system and the Material
+Design color system. Most components have a themer for at least one of these systems.
+
+### Sensible defaults, yet highly configurable
```
Comment (romoore) [inline docs/theming/README.md:18]: This should be an `H2` since it is sitting directly within an `H1`.

**Review by romoore — APPROVED** [review]
Structure/style suggestions.

File: docs/theming/README.md (hunk 3 above)
Comment (jverkoey) [reply docs/theming/README.md:18]: I'm attempting to make this block part of the introductory text, but `h2` gives a visual appearance of separation. Open to suggestions for alternative solutions here.

**Pull request merged**